src/python/pants/jvm/resources.py

```diff
@@ -119,8 +118,6 @@ async def assemble_resources_jar(
     )
 
     output_digest = resources_jar_result.output_digest
-    if jvm.reproducible_jars:
```

Comment:
Should the option be deprecated?

Reply:
it is still in use for java and scala jars, just not needed anymore for resources jar.